examples/bin/start-druid-main.py

```diff
@@ -436,43 +428,39 @@ def check_memory_constraint(total_memory, services):
     return int(total_memory * 0.8)
 
 
-def build_mm_task_java_opts_array(memory_type):
-    task_memory = '-D{0}=['.format(TASK_JAVA_OPTS_PROPERTY)
-
-    mem_array = TASK_MEM_MAP.get(memory_type)
+def build_mm_task_java_opts_array(task_memory):
+    tasks_memory = '-D{0}=['.format(TASK_JAVA_OPTS_PROPERTY)
```

Comment:
The similar names `task_memory`, `tasks_memory` seem a little confusing. Maybe rename this field to `task_java_opts_string` or `task_java_opts_value`.